runtime/doc/remote_plugin.txt

```diff
@@ -15,34 +15,34 @@ Nvim support for remote plugins 		  *remote-plugin*
 1. Introduction					    *remote-plugin-intro*
 
 Extensibility is a primary goal of Nvim. Any programming language may be used
-to extend nvim without changes to nvim itself. This is achieved with remote
+to extend Nvim without changes to Nvim itself. This is achieved with remote
 plugins, coprocesses that have a direct communication channel (via
 |msgpack-rpc|) with the Nvim process.
 
-Even though these plugins are running in separate processes they can call, be
-called, and receive events just as if the code was being executed in the main
+Even though these plugins run in separate processes they can call, be called,
+and receive events just as if the plugin's code were executed in the main
 process.
 
+
```

Comment:
An extra line such as this doesn't look to be present anywhere else in this document, so should probably be removed.